software/receipt-wrangler.yml

```
@@ -0,0 +1,14 @@
+name: "Receipt Wrangler"
+website_url: "https://receiptwrangler.io"
+source_code_url: "https://github.com/Receipt-Wrangler"
+description: "Open Source, Self hosted, Receipt Management."
```

```suggestion
description: " An easy to use Receipt manager. Allowing users to create receipts effortlessly and quickly, categorize and more."
```
This should be a better description, feel free to correct others 👍🏻

---

👍👍 Thanks for the suggestion. Updated!